### ISSUE TITLE
"Uncaught ReferenceError" in browser's JavaScript console

### ISSUE THREAD

**Issue by markusschuh** [devonfw/my-thai-star#297]
### Expected behavior

As a developer, I want the my-thai-star frontend to not output errors in browser's JavaScript console out of the box so that further development is easier.

### Actual behavior

On access to http://localhost:4200 the browser console shows

```
index.js:2 Uncaught ReferenceError: global is not defined
    at Object../node_modules/intl/index.js (index.js:2)
    at __webpack_require__ (bootstrap:79)
    at Module../src/polyfills.ts (polyfills.ts:1)
    at __webpack_require__ (bootstrap:79)
    at Object.2 (polyfills.ts:68)
    at __webpack_require__ (bootstrap:79)
    at checkDeferredModules (bootstrap:45)
    at Array.webpackJsonpCallback [as push] (bootstrap:32)
    at polyfills.js:1
```
where index.js is from node_modules/intl/

### Steps to reproduce (bug) / Use Case of feature request (enhancement)

1. git clone my-thai-star  (default branch `develop` as of today)
1. yarn
1. ng serve
1. open browser and JavaScript Console in its web developer tools
1. access http://localhost:4200

### Comments/Hints:

According to https://github.com/angular/angular-cli/issues/9920 the `intl` dependency is no longer needed and should be removed.

### Affected version:

* OS: Windows 10
* Browser: Chrome Version 77.0.3865.120 (Official Build) (64-bit)

**Comment by markusschuh** [comment]
Related to #172, where the change was introduced.

**Comment by markusschuh** [comment]
Fixed by intl remove within #309
